spec/controllers/api/drafts_controller_spec.rb

```diff
@@ -79,6 +79,21 @@
     assert_equal(parsed_body['error'], 'unauthorized')
   end
 
+  it 'can not update if the user does not belong to the provider list.' do
+    # The draft is created by a MMT_2 user
+    # The user updating the document does not have MMT_2 in their provider list, only 'LARC'
+    allow_any_instance_of(Cmr::UrsClient).to receive(:validate_mmt_token).and_return(Faraday::Response.new(status: 200, body: '{"uid":"testuser2"}', response_headers: { 'Content-Type': 'application/json; charset=utf-8' }))
```

Comment:
I would just make a note in the comment that testuser2's available provider list is just LARC.

Reply:
fixed